pkg/discovery/dtofactory/application_entity_dto_builder.go

```diff
@@ -107,6 +107,11 @@ func (builder *applicationEntityDTOBuilder) BuildEntityDTOs(pods []*api.Pod) ([]
 				ebuilder.Monitored(false)
 			}
 
+			truep := true
+			ebuilder.ConsumerPolicy(&proto.EntityDTO_ConsumerPolicy{
```

Comment:
Why are we making app mandatory provider true?

Reply:
There is no longer a mandatory provider.  If providerMustClone is true, then the underlying provider will clone if the current SE needs to clone.

Reply:
Sorry for the confusion, plz ignore my comment here and the one below!